### PR TITLE
[RHCLOUD-30332] Add rbac-cache metrics and init log

### PR DESCRIPTION
**Pull request by g-duval**

### CODE REVIEW

File: backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -49,6 +49,8 @@ internal.admin-role=crc-notifications-team
 
 # Duration rbac entries are kept in cache
 quarkus.cache.caffeine.rbac-cache.expire-after-write=PT120s
+quarkus.cache.caffeine.rbac-cache.metrics-enabled=true
+quarkus.log.category."io.quarkus.cache.runtime.caffeine.CaffeineCacheManagerBuilder".level=DEBUG
```
Comment (g-duval) [inline backend/src/main/resources/application.properties:53]: Just to be sure than cache setup values are used.
Could be removed later.

Comment (gwenneg) [reply backend/src/main/resources/application.properties:53]: Do you know if this could spam the logs with tons of entries?

Comment (g-duval) [reply backend/src/main/resources/application.properties:53]: I tested it localy, it just add 2 lines at startup like:
`2024-02-06 14:26:44,651 DEBUG [io.qua.cac.run.caf.CaffeineCacheManagerBuilder] (Quarkus Main Thread) Building Caffeine cache [maintenance] with [initialCapacity=null], [maximumSize=null], [expireAfterWrite=PT1M], [expireAfterAccess=null] and [metricsEnabled=false]
2024-02-06 14:26:44,654 DEBUG [io.qua.cac.run.caf.CaffeineCacheManagerBuilder] (Quarkus Main Thread) Building Caffeine cache [rbac-cache] with [initialCapacity=null], [maximumSize=null], [expireAfterWrite=PT2M], [expireAfterAccess=null] and [metricsEnabled=true]
`

Comment (gwenneg) [reply backend/src/main/resources/application.properties:53]: My bad, I misread the conf key. This one indeed cannot spam the logs 👍

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**